kairon/shared/concurrency/actors/analytics_runner.py

```
@@ -81,14 +82,26 @@ def execute(self, source_code: Text, predefined_objects: Optional[Dict] = None,
             return self.__cleanup(result)
 
         except Exception as e:
             msg = stdout.strip() if 'stdout' in locals() and stdout else str(e)
             logger.exception(msg)
-            raise AppException(f"Execution error: {msg}") from e
+            triggers = action.get("triggers", [])
+            for trigger in triggers:
+                if trigger.get("conditions") == TriggerCondition.failure.value and trigger.get("action_type") == "email_action":
+                    try:
+                        AnalyticsPipelineProcessor.trigger_email(action, bot)
+                    except Exception:
```

Ruff (0.14.10)

Do not catch blind exception: `Exception`

(BLE001)